mkt/reviewers/tests/test_views_api.py

```diff
@@ -301,6 +302,34 @@ def test_no_premium_filtering(self):
         eq_(len(res.json['objects']), 1)
 
 
+class TestWebsiteReviewerActions(RestOAuth):
```

Comment:
Can you add 2 small permissions tests ? Like one with an anonymous user, one with a logged in user with no permissions